lib/helper/WebDriverIO.js

```diff
@@ -924,7 +1002,71 @@ class WebDriverIO extends Helper {
       }).then((selected) => {
         if (!Array.isArray(selected)) selected = [selected];
         selected = selected.filter((val) => val === true);
-        return assert.equal(selected.length, num, `expected nummber of visible elements (${locator}) is ${num}, but found ${selected.length}`);
+        return assert.equal(selected.length, num,
+          `expected number of visible elements (${locator}) is ${num}, but found ${selected.length}`);
+      });
+    });
+  }
+
+  /**
+   * Checks that there is an Element with given CSS property.
+   */
+  seeCssPropertyOnElement(locator, cssProperty, value) {
+    let client = this.browser;
+    return this._locate(locator).then((res) => {
+      if (!res.value || res.value.length === 0) {
+        throw new Error(`${locator} cant be located by name|text|CSS|XPath`);
+      }
+      let commands = [];
+      res.value.forEach((el) => commands.push(client.elementIdCssProperty(el.ELEMENT, cssProperty)));
+      return client.unify(commands, {
+        extractValue: true
+      }).then((selected) => {
+        if (!Array.isArray(selected)) selected = [selected];
+        selected = selected.filter((val) => val === value);
+        return assert.ok(selected.length > 0,
+          `Element ${locator} hasn't CSS property ${cssProperty} with value ${value}`);
+      });
+    });
+  }
+
+  /**
+   * Checks that there is an Element with given attribute.
+   */
+  seeAttributeOnElement(locator, attr, value) {
```

Comment:
actually, this can be merged with `seeElement`, with optional second parameter.
signature can be simplified:

```js
I.seeElement('a', ['href']); // seeks for an element with href attribute
I.seeElement('a', ['href', 'target']); // seeks for an element with href and target
I.seeElement('a', { target: '_blank' } ); // element with attribute target=_blank
```